programs/hash/hello.c

```diff
@@ -48,7 +48,7 @@ int main( void )
 #else
 int main( void )
 {
-    int i, ret;
+    int i, ret = MBEDTLS_EXIT_SUCCESS;
```

Comment:
I'd rather opt for initializing return codes with invalid values, but that's a minor.